README.md

```diff
@@ -29,10 +29,11 @@ This is my research and reflection journal for DGL 104 winter 2023
 
 # Reflection of Style guide
 
+**REDO THIS**
+
```

Comment:
I appreciate that you addressed this reflection, but in reading the below it looks like you cut and pasted it from elsewhere. And it doesn't really answer the question. Please redo this in your own words.